common/test/acceptance/tests/lms/test_teams.py

```diff
@@ -455,3 +456,199 @@ def test_teams_membership(self):
             self.browse_teams_page.team_cards[0].find_element_by_css_selector('.member-count').text,
             '1 / 10 Members'
         )
+
+    def test_navigation_links(self):
+        """
+        Scenario: User should be able to navigate to "browse all teams" and "search team description" links.
+        Given I am enrolled in a course with a team configuration and a topic
+            containing one team
+        When I visit the Teams page for that topic
+        Then I should see the correct page header
+        And I should see the link to "browse all team"
+        And I should navigate to that link
+        And I see the relevant page loaded
+        And I should see the link to "search teams"
+        And I should navigate to that link
+        And I see the relevant page loaded
+        """
+        self.browse_teams_page.visit()
+        self.verify_page_header()
+
+        self.browse_teams_page.wait_for_action_links_tobe_present()
+        self.browse_teams_page.click_browse_all_teams_link()
+        self.assertTrue(self.topics_page.is_browser_on_page())
+
+        self.browse_teams_page.visit()
+        self.verify_page_header()
+        self.browse_teams_page.wait_for_action_links_tobe_present()
+        self.browse_teams_page.click_search_team_link()
+        # TODO Add search page expectation once that implemented.
+
+
+@attr('shard_5')
+class CreateTeamTest(TeamsTabBase):
+    """
+    Tests for creating a new Team within a Topic on the Teams page.
+    """
+
+    def setUp(self):
+        super(CreateTeamTest, self).setUp()
+        self.topic = {'name': 'Example Topic', 'id': 'example_topic', 'description': 'Description'}
+        self.set_team_configuration({'course_id': self.course_id, 'max_team_size': 10, 'topics': [self.topic]})
+        self.browse_teams_page = BrowseTeamsPage(self.browser, self.course_id, self.topic)
+        self.browse_teams_page.visit()
+        self.create_team_page = CreateTeamPage(self.browser, self.course_id, self.topic)
+        self.team_name = 'Avengers'
+
+    def verify_page_header(self):
+        """
+        Verify that the page header correctly reflects the
+        create team header, description and breadcrumb.
+        """
+        self.assertEqual(self.create_team_page.header_page_name, 'Create a New Team')
+        self.assertEqual(
+            self.create_team_page.header_page_description,
+            'Create a new team if you can\'t find existing teams to join, '
+            'or if you would like to learn with friends you know.'
+        )
+        self.assertEqual(self.create_team_page.header_page_breadcrumbs, self.topic['name'])
+
+    def verify_and_navigate_to_create_team_page(self):
```

Comment:
I think this method is unnecessary. The first line provides no value, since you only ever call this when you're on the browse teams page. The second line should go away once you implement the change I recommended above. Finally, the last step has nothing to do with navigating to the create team page, and would be better off in the test method itself.